doc/cmp.txt

```diff
@@ -686,12 +687,21 @@ view.docs.auto_open~
 
   Specify whether to show the docs_view when selecting an item.
 
-                                                 *cmp-config.view.follow_cursor*
-view.follow_cursor*
+                                                *cmp-config.view.entries.follow_cursor*
```

Comment:
it should be selection_order?

Reply:
I've added selection_order above this as a separate entry. `config.view.entries.follow_cursor` is the proper name for the option added by my PR yesterday, but `config.view.entries.selection_order` is another option that is already available to the user but is undocumented in this section as of right now.